test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -5746,6 +5746,34 @@ public async Task ValidateExtensionLifecycleLogs()
             JObject.Parse(configJson);
         }
 
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task TestStoreInputsInOrchestrationHistory()
+        {
+            DurableTaskOptions options = new DurableTaskOptions();
+            options.StoreInputsInOrchestrationHistory = true;
+
+            using (var host = TestHelpers.GetJobHostWithOptions(
+                   this.loggerProvider,
+                   options))
+            {
+                await host.StartAsync();
+                var client1 = await host.StartOrchestratorAsync(nameof(TestOrchestrations.SayHelloWithActivity), "foo", this.output);
+                var client2 = await host.StartOrchestratorAsync(nameof(TestOrchestrations.CallActivityWithNoInput), null, this.output);

```

Comment:
One way to do this is just to put both activities in the same orchestrator and check input for both history results. But I wont block on this.